purchase_work_acceptance_evaluation/tests/test_purchase_work_acceptance_evaluation.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields
+from odoo.exceptions import ValidationError
+from odoo.tests.common import Form, TransactionCase
+
+
+class TestPurchaseWorkAcceptanceEvaluation(TransactionCase):
+    def setUp(self):
+        super(TestPurchaseWorkAcceptanceEvaluation, self).setUp()
+        # Create Vendor
+        self.res_partner = self.env.ref("base.res_partner_3")
+        # Create Employee
+        self.employee = self.env.ref("base.user_demo")
+        # Create Date
+        self.date_now = fields.Datetime.now()
+        # Create Score
+        self.name = self.env.ref("purchase_work_acceptance_evaluation.score_on_time")
+        # Create Case
+        self.evaluation_id = self.env.ref("purchase_work_acceptance_evaluation.case_name_01")
+
+    def test_00_wa_button(self):
+        work_acceptance = self.env["work.acceptance"].create(
```

Comment:
If possible, you can explain what the case did. Otherwise, name self explained function. I.e., 
```
def test_wa_check_state_required(self):
```